redistributable/SpringBoot/generated.iml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks like a generated file. Those should not be submitted to source control.